DOCUMENTATION.md

```diff
@@ -1,26 +1,41 @@
-## Table of Contents
+# Table of Contents
 * [Description](#description)
 * [MQTTClient](#mqttclient)
-* [connect](#connect)
-* [close](#close)
-* [set_uplink_callback](#set_uplink_callback)
-  * [uplink_callback](#uplink_callback)
-* [set_connect_callback](#set_connect_callback)
-  * [connect_callback](#connect_callback)
-* [set_downlink_callback](#set_downlink_callback)
-  * [downlink_callback](#downlink_callback)
-* [set_close_callback](#set_close_callback)
-  * [close_callback](#close_callback)
-* [send](#send)
+  * [connect](#connect)
+  * [close](#close)
+  * [set_uplink_callback](#set_uplink_callback)
+    * [uplink_callback](#uplink_callback)
+  * [set_connect_callback](#set_connect_callback)
+    * [connect_callback](#connect_callback)
+  * [set_downlink_callback](#set_downlink_callback)
+    * [downlink_callback](#downlink_callback)
+  * [set_close_callback](#set_close_callback)
+    * [close_callback](#close_callback)
+  * [send](#send)
+  * [UplinkMessage](#uplinkmessage)
+* [ApplicationClient](#applicationclient)
+  * [get](#get)
+  * [set_payload_format](#set_payload_format)
+  * [set_custom_payload_functions](#set_custom_payload_functions)
+  * [set_register_on_join_access_key](#set_register_on_join_access_key)
+  * [unregister](#unregister)
+  * [device](#device)
+  * [devices](#devices)
+  * [update_device](#update_device)
+  * [delete_device](#delete_device)
+  * [Device](#device)
+  * [Application](#application)
+* [HandlerClient](#handlerclient)
+  * [data](#data)
+  * [application](#application)
 * [Errors](#errors)
-* [UplinkMessage](#uplinkmessage)
 
-## Description
 
-This package provides you an easy way to connect to The Things Network via MQTT.
+## Description
+This package provides you an easy way to connect to The Things Network via MQTT or manage your applications.
```

Comment:
I would have written: *to ~connect to~ **exchange traffic with** The Things Network ~or~ **and** to manage your applications.*